packages/web-app-files/src/components/ActionMenuItem.vue

```diff
@@ -3,8 +3,10 @@
     <component
       :is="action.componentType"
       v-bind="getComponentProps(action, items)"
-      :class="['oc-text-bold', action.class]"
+      :class="[action.class, 'action-menu-item']"
       data-testid="action-handler"
+      appearance="raw"
```

Comment:
This defaults to `raw` in the ActionMenuItem's props already, why hardcode it here?

Reply:
i think it was by mistake